modular_skyrat/modules/icemoon_additions/code/wolf.dm

```diff
@@ -0,0 +1,108 @@
+/datum/pet_command/good_boy/wolf
+	speech_commands = list("good wolf")
+
+/datum/pet_command/follow/wolf
+	// Nordic-themed for a bit of extra flavor
+	speech_commands = list("heel", "follow", "fylgja", "fyl")
```

Comment:
I had to google what fylgja meant but uh, lol? 
![image](https://github.com/Skyrat-SS13/Skyrat-tg/assets/22140677/2211cbb5-010f-42c1-b719-1ea8d654dfc7)

Is that meant to be an alternate to follow?

Reply:
yeah, the idea is for the icecats (and nordic chuunibyou assistants) to scream FYL at the top of their lungs when directing their pack of 12 wolves

Reply:
neat lil flavortext - guessing the oil is just an icecat version of the pet tag?